code/Player/Cameras/FreeSpectateCamera.cs

```diff
@@ -62,5 +62,19 @@ public override void BuildInput( InputBuilder input )
 		_lookAngles.roll = 0;
 
 		base.BuildInput( input );
+
+		if ( input.Pressed( InputButton.Use ) )
+			FindTargetProp();
+	}
+
+	private void FindTargetProp()
+	{
+		var lookDir = Rotation.From( _lookAngles ).Forward;
+		var tr = Trace.Ray( Position, Position + (lookDir * 100f) )
+			.EntitiesOnly()
+			.Run();
+		
+		if ( tr.Entity is Prop p && p.PhysicsBody is not null )
+			ConsoleSystem.Run( "ttt_possess_prop", p.NetworkIdent );
```

Comment:
You can just call the method itself, you don't need the `ConsoleSystem.Run` stuff, there are examples in `C4` I think in our repo.